generic/tests/cfg/netperf.cfg

```diff
@@ -55,8 +55,7 @@
     #Test base env configration
     ver_cmd = rpm -q qemu-kvm
     netperf_version = 2.6.0
-    netperf_download_link = ftp://ftp.netperf.org/netperf/archive/netperf-2.6.0.tar.bz2
-    pkg_md5sum = 9654ffdfd4c4f2c93ce3733cd9ed9236
+    netperf_pkg = performance/netperf-2.6.0.tar.bz2
```

Comment:
This is ok.
But I think put `performance/` into line 99, leave it just `netperf-2.6.0.tar.bz2` could be better.

Reply:
After discuss with ＠hereischen, I will not modify here, it's ok.